MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
+    logrotate:
+        daysToKeep: 90
+        #numToKeep: 20
+        artifactDaysToKeep: 30
+        #artifactNumToKeep: 10
```

Comment:
This is not needed - if we find any need to define a default logrotate settings in the future, let's add it. Right now it's just a guess and therefore not very useful. It just adds unimportant clutter.